README.md

```diff
@@ -1,13 +1,20 @@
-# whitehall
-
-### Running the Application
-
-**Use GOV.UK Docker to run any commands that follow.**
+# Whitehall
 
 Whitehall is deployed in two modes:
 
 - 'admin' for publishers to create and manage content (e.g. <http://whitehall-admin.dev.gov.uk/government/admin/news/new>)
-- 'frontend' for rendering content (eg: <http://www.gov.uk/government/get-involved>, <https://www.gov.uk/world>)
+- 'frontend' for rendering content (eg: <https://www.gov.uk/government/get-involved>, <https://www.gov.uk/world>)
```

Comment:
How about: 

_'frontend' for rendering some legacy content (see live examples that follow)_

This both stops us repeating ourselves with those two URLs and wraps up the issue you were discussing previously.

Reply:
It was mentioned: 

> It does open a bit of a can of worms to determine what the legacy status is

I've updated as suggested